tests/unit/dojson/test_dojson_experiments.py

```diff
@@ -29,210 +30,212 @@
 from inspirehep.dojson.utils import validate
 
 
-def test_contact_details_from_marcxml_270_single_p_single_m():
+def test_experiment_dates_from_marcxml_046():
     schema = load_schema('experiments')
-    subschema = schema['properties']['contact_details']
+    subschema_q = schema['properties']['date_proposed']
+    subschema_r = schema['properties']['date_approved']
+    subschema_s = schema['properties']['date_started']
+    subschema_t = schema['properties']['date_completed']
+    subschema_c = schema['properties']['date_cancelled']
 
     snippet = (
-        '<datafield tag="270" ind1=" " ind2=" ">'
-        '  <subfield code="m">lindner@mpi-hd.mpg.de</subfield>'
-        '  <subfield code="p">Manfred Lindner</subfield>'
-        '</datafield>'
+        '<record>'
+        '  <datafield tag="046" ind1=" " ind2=" ">'
+        '    <subfield code="q">2014</subfield>'
+        '    <subfield code="r">2015</subfield>'
+        '    <subfield code="s">1995</subfield>'
+        '    <subfield code="t">2008</subfield>'
+        '    <subfield code="c">2016</subfield>'
+        '  </datafield>'
+        '</record>'
```

Comment:
This test is way too optimistic: you'll never find data this clean in INSPIRE legacy. Look at what's actually in the records!